docs/use-cases/jobsearch-agent.md

```diff
@@ -6,7 +6,7 @@ It utilize various tools for internet search and document comparison to fulfill
 
 First, we need to import the necessary modules. Each module serves a specific purpose in our script. We utilize various tools for internet search and document comparison to fulfill the agent's task. Here’s what each import does:
 
-* `GoogleSerpAPISearch` and `DuckDuckGoSearch` are tools for performing web searches.
+* `SearchApiSearch`, `GoogleSerpAPISearch` and `DuckDuckGoSearch` are tools for performing web searches.
```

Comment:
thank you for the PR. Could you please revert your changes for the use cases?

Rather please create a cookbook for showcasing the API